cpython-unix/build-cpython.sh

```diff
@@ -427,7 +427,11 @@ if [ -n "${CPYTHON_OPTIMIZED}" ]; then
 fi
 
 if [ -n "${CPYTHON_LTO}" ]; then
-    CONFIGURE_FLAGS="${CONFIGURE_FLAGS} --with-lto"
+    # On Python 3.12 and 3.13, `--with-lto` enables ThinLTO by default, while on other versions it
```

Comment:
What about 3.14? Did it change in 3.14?

Reply:
3.14a5 and below still defaults to ThinLTO. 3.14a6 (upcoming) will default back to fullLTO.

It's safe to just pass `--with-lto=full` to all versions of 3.14. That will always work.

Reply:
Yes the linked documentation says they're reverting the ThinLTO default in 3.14.

Seems best to just set it on all versions.